documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md

```diff
@@ -0,0 +1,136 @@
+* [ ] Chris Howell - (Vonage) Active Maintainer
+* [x] Diego González Martínez - (Telefónica) Active Maintainer
+* [x] Doug Makishima - (Summit Tech) End User Council representative
+* [ ] George Glass - TM Forum representative
+* [ ] Henry Calvert - GSMA representative
+* [x] Mark Cornall - GSMA representative
+* [x] Jan Friman - (Ericsson) Active Maintainer
+* [x] Jose Luis Urien - (Telefonica) Active Maintainer
+* [ ] Kevin Smith - (Vodafone) Active Maintainer
+* [x] Mahesh Chapalamadugu - (Verizon) End User Council representative
+* [x] Shilpa Padgaonkar - (Deutsche Telekom AG) Active Maintainer
+* [x] Toshi (Toshiyasu) Wakayama - (KDDI) Active Maintainer
+
+Guest Presenter:
+
+* Dawid McBride (Linux Fundation)
+
+Attendees:
+*  Sylvain Morel (Orange)
+*  Helene Vigue (GSMA)
+*  Ivan Izaquirre MArtin (Telefonica)
+*  Dinesh Muthukumar
+*  Mehdi Jafarizadeh
+*  Pierre Close (AT&T)
+*  Casey Cain (Linux Fundation)
+*  Tanja de Groot (Nokia)
+
+
+Minutes are available within: [https://github.com/camaraproject/Governance/tree/main/documentation/MeetingMinutes/TSC](https://github.com/camaraproject/Governance/tree/main/documentation/MeetingMinutes/TSC)
+
+## Agenda
+
+* Introduction
+* Release Management (#82) 
+* Commonalities & ICM 
+    * Alignment around securitySchemes and security of CAMARA API specs
+* API Backlog 
+    * New proposal for review "Network Insights"
+    * Decision on API Family for "Device Swap"
+    * Status of Site-to-cloud VPN proposal
+* (Further) open issues in https://github.com/camaraproject/Governance/issues
+* AOB
+
+Please find material for the agenda points below.
+
+## Introduction
+
+The participation of TSC Members covers well all CAMARA subproject.
+
+## Release Management
+
+A deck was prepared by Ludovic/Herbert (Available within meeting notes folder)
+
+Following points were discussed:
+
+**Discussion on API versioning** – DT proposal to use alpha<n> in API path until and API (major) version is stable
+  * Distinction between repostiory release (semantic version as before) and the version shown within API base path
+    * A sub project repository can contain one or multiple (closely related) APIs
+    * The version number of releases on sub project level will be used within all APIs within the repository (= the API release belongs to vX.Y.Z of the sub project)
+    * In addition every API within a sub project will have in the basepath
+      * the major version of the API only in case of a **stable** API (e.g. qod/v1)
+      * the major version of the API + a postfix alpha or beta, potentially with a number (e.g. qod/v1alpha3 for the third alpha version of an API)
+      * (potentially only alpha and stable will be used within CAMARA sub projects)
+    * A sub project (release) can contain
+      *  multiple APIs with different maturity, e.g. location-verification/v1 and location-retrieval/v1alpha1
+      *  multiple versions of the same API, e.g. edge-discovery/v1 (stable) and edge-discovery/v2alpha1

```

Comment:
I also noted down we indicated that sub project release will remain with 0.x until first API reaches stable v1. E.g.: A release v0.1 may include an API in v1alphaX
Anyway we should further discuss on these details, once agreements are written down, small details may arise

Reply:
Agreed - added.

Reply:
I suppose we need a more elaborated concept to discuss the details. The sentence was originally meant that there will be full semantic versioning only on the release of the sub project. 

But indeed, such release can contain APIs which have different major versions within the base path. To give an example:

Release 2.x.y of Device Location can contain (more complex then it most probably will be in practice, just to show the principles):
- location-verification/v1 (the first stable version, if still supported, not anymore evolved)
- location-verification/v2 (the second stable version, due to breaking changes a different API)
- location-verification/v2alpha3 (alpha version with enhancements compared to v2 which potentially will be added to the stable version within a next release of DeviceLocation)
- location-retrieval/v1 (stable version)
- location-retrieval/v2alpha1 (first alpha of next major version of location-retrieval, new major number due to lack of backward-compatibility to location-retrieval/v1)
All these API versions are belonging to the same release of the repository DeviceLocation and are unambigious defined by the commit the release is pointing to.

Reply:
yes, agreed.